vector/src/main/java/im/vector/app/features/home/ShortcutCreator.kt

```diff
@@ -33,6 +33,7 @@ import javax.inject.Inject
 private val useAdaptiveIcon = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
 private const val adaptiveIconSizeDp = 108
 private const val adaptiveIconOuterSidesDp = 18
+private const val directShareCategory = "im.vector.app.SHORTCUT_SHARE"
```

Comment:
It should be better to use `BuildConfig.APPLICATION_ID` as a prefix here, but will it be possible in the file `shortcuts.xml`?

Reply:
Is there a way to get the `applicationId` with the suffix in the `build.grade` or the `BuildConfig.APPLICATION_ID` without the suffix? With suffix, I mean that `.debug` at the end.

Reply:
Thinking again about it, I think the only way is to have a file `shortcuts.xml` in the release folder and one in the debug folder. In the code, we can use `BuildConfig.APPLICATION_ID`